app/controllers/home_controller.rb

```diff
@@ -0,0 +1,14 @@
+class HomeController < ApplicationController
+  before_action :authenticate_student!, only: [:students]
+  before_action :authenticate_instructor!, only: [:instructors]
+  def index
+  end
```

Comment:
Add redirect student to student dashboard and instructor to instructor dashboard